sirepo/raydata/analysis_driver/__init__.py

```diff
@@ -105,3 +131,10 @@ def init(catalog_names):
             importlib.import_module(f"sirepo.raydata.analysis_driver.{n}"),
             n.upper(),
         )
+
+
+_cfg = pkconfig.init(
+    conda_prefix=pkconfig.RequiredUnlessDev(
```

Comment:
I don't think this needs to be RequiredUnlessDev. Currently [we use this path in > dev](https://github.com/radiasoft/raydata/blob/da9a2ed3c038728319129d8ba2ef29442a764fec/container-conf/build.sh#L7).

There is potential that we'll end up using a conda_prefix mounted in the container but for now ~/miniconda is fine.

Reply:
a4175ab0127a5b2207443ee18cab9d384c401eb9